### ISSUE TITLE
login issues

### ISSUE THREAD

**Issue by codingbulljunky** [bwilczynski/strava-cli#3]
When I try to login, it takes me to the right page but after I click 'authorize', I return to the terminal where I get the following error. Any ideas?

```
Traceback (most recent call last):
  File "/usr/local/bin/strava", line 11, in <module>
    load_entry_point('strava-cli==0.4.9', 'console_scripts', 'strava')()
  File "/usr/local/Cellar/strava-cli/0.4.9/libexec/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/Cellar/strava-cli/0.4.9/libexec/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/Cellar/strava-cli/0.4.9/libexec/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/Cellar/strava-cli/0.4.9/libexec/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/Cellar/strava-cli/0.4.9/libexec/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/Cellar/strava-cli/0.4.9/libexec/lib/python3.7/site-packages/strava/decorators.py", line 86, in wrapper_config_required
    return func(*args, **kwargs)
  File "/usr/local/Cellar/strava-cli/0.4.9/libexec/lib/python3.7/site-packages/strava/commands/login.py", line 20, in login
    data = auth_flow.get_access_token(code)
  File "/usr/local/Cellar/strava-cli/0.4.9/libexec/lib/python3.7/site-packages/strava/api/oauth2.py", line 64, in get_access_token
    client_secret=self.client_secret)
  File "/usr/local/Cellar/strava-cli/0.4.9/libexec/lib/python3.7/site-packages/requests_oauthlib/oauth2_session.py", line 307, in fetch_token
    self._client.parse_request_body_response(r.text, scope=self.scope)
  File "/usr/local/Cellar/strava-cli/0.4.9/libexec/lib/python3.7/site-packages/oauthlib/oauth2/rfc6749/clients/base.py", line 415, in parse_request_body_response
    self.token = parse_token_response(body, scope=scope)
  File "/usr/local/Cellar/strava-cli/0.4.9/libexec/lib/python3.7/site-packages/oauthlib/oauth2/rfc6749/parameters.py", line 425, in parse_token_response
    validate_token_parameters(params)
  File "/usr/local/Cellar/strava-cli/0.4.9/libexec/lib/python3.7/site-packages/oauthlib/oauth2/rfc6749/parameters.py", line 435, in validate_token_parameters
    raise MissingTokenError(description="Missing access token parameter.")
oauthlib.oauth2.rfc6749.errors.MissingTokenError: (missing_token) Missing access token parameter.
```

**Comment by luisnabais** [comment]
Hello.

I have exactly the same issue.

Python 3.8.0.
strava-cli 0.4.9

**Comment by bwilczynski** [comment]
Should be solved with #2

**Comment by codingbulljunky** [comment]
I've re-installed, but still running into the issue above. I'm not very familiar with python, what exactly should I do to fix it?
